Automattic/jetpack
# Sync Calypso locale to Atomic Classic & Atomic Default

Part of https://github.com/Automattic/dotcom-forge/issues/8835

## Proposed changes:


This PR changes to sync the Calypso locale to Atomic Classic and Atomic Default. This ensures that WordPress.com's locale is always used in wp-admin and the Calypso page.

- The original code is in: https://github.com/Automattic/jetpack/blob/a950c3f0172479cd01e8a55ad030bb309684b80f/projects/packages/masterbar/src/masterbar/class-masterbar.php#L139-L144
- https://github.com/Automattic/jetpack/pull/38419/ caused regressions, so the code was not being loaded. This PR also fixes that issue. 

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






Atomic Default
* Prepare an Atomic Default
* Patch this PR
* Go to any wp-admin page
* Observe the language you set in /me/account is set 

Atomic Classic
* Repeat the above steps in Atomic Classic

Simple sites
* Ensure this does not change the behavior in Simple sites
